File: backend/MediaVerse/MediaVerse.Domain/Entities/Album.cs
namespace MediaVerse.Domain.Entities;

public partial class Album
{
    public Guid Id { get; set; }

    public virtual Entry IdNavigation { get; set; } = null!;

    public virtual ICollection<MusicGenre> MusicGenres { get; set; } = new List<MusicGenre>();

    public virtual ICollection<Song> Songs { get; set; } = new List<Song>();
}

File: frontend/src/models/entry/album/Album.ts
import { Entry } from '../Entry.ts'
import { WorkOn } from '../WorkOn.ts'
import { EntryPreview } from '../../author/Author.ts'

export interface Album {
  entry: Entry,
  musicGenres: string[],
  workOns: WorkOn[],
  songs: EntryPreview[]
}
